apps/journeys-admin/src/components/Editor/Drawer/Drawer.tsx

```diff
@@ -96,7 +102,7 @@ export function Drawer(): ReactElement {
         open={mobileOpen}
         onClose={handleDrawerToggle}
         sx={{
-          display: { xs: 'block', sm: 'none' }
+          display: { xs: 'block', sm: 'block' }
```

Comment:
Does this need to be `display: none` for `md`? If not it should just be 
```suggestion
          display:  'block'
```